src/app/components/applications/SSV/MyAccount/components/Validator/SingleCluster/SingleCluster.tsx

```diff
@@ -76,17 +74,23 @@ const SingleCluster = () => {
     notificationsStore.showMessage('Copied to clipboard.', 'success');
   };
 
-  const rows = clusterValidators?.map((validator: any)=>{
-    return createData(
-        <Grid container style={{ alignItems: 'center', gap: 8 }}>
-          <Grid item>0x{longStringShorten(validator.public_key, 4)}</Grid>
-          <ImageDiv onClick={() => copyToClipboard(validator.public_key)} image={'copy'} width={24} height={24} />
-        </Grid>,
-        <Status item={validator} />,
-        <Grid item>33.12 ETH</Grid>,
-        <Grid item>{12.3}%</Grid>,
-    );
-  });
+  const sortValidatorsByStatus = () => {
+    setClusterValidators(prevState => [...prevState.sort((a: any, b: any) => a.status === b.status ? 0 : a.status ? -1 : 1)]);
+  };
+
+  useEffect(() =>{
+      setRows(clusterValidators?.map((validator: any)=>{
```

Comment:
This question mark doesn't make sense here because if it will be null or undefined, this line will raise exception with reason that null or undefined doesn't have `map` method.

To fix that you can wrap it will default empty array: `(clusterValidators || []).map...`